src/SFML/Graphics/Text.cpp

```diff
@@ -446,15 +441,14 @@ void Text::ensureGeometryUpdate() const
     bool  isBold             = m_style & Bold;
     bool  isUnderlined       = m_style & Underlined;
     bool  isStrikeThrough    = m_style & StrikeThrough;
-    float italicShear        = (m_style & Italic) ? sf::degrees(12).asRadians() : 0.f;
+    float italicShear        = (m_style & Italic) ? degrees(12).asRadians() : 0.f;
     float underlineOffset    = m_font->getUnderlinePosition(m_characterSize);
     float underlineThickness = m_font->getUnderlineThickness(m_characterSize);
 
     // Compute the location of the strike through dynamically
     // We use the center point of the lowercase 'x' glyph as the reference
     // We reuse the underline thickness as the thickness of the strike through as well
-    FloatRect xBounds             = m_font->getGlyph(U'x', m_characterSize, isBold).bounds;
-    float     strikeThroughOffset = xBounds.top + xBounds.height / 2.f;
+    float strikeThroughOffset = m_font->getGlyph(U'x', m_characterSize, isBold).bounds.getCenter().x;
```

Comment:
This should be the `bounds.getCenter().y` instead of x. The comment above details how they're getting the centre point of the letter 'x'. That is why the bounds was called `xBounds`. Using the `.x` will produce the wrong offset.

Reply:
Accidental. Thank you